src/README.md

```diff
@@ -1206,6 +1228,51 @@ it, this can happen at any time after the garbage collector notices that
 it's no longer reachable and before the V8 isolate is torn down.
 See the [Oilpan documentation in Chromium][] for more details.
 
+When a cppgc-managed object is no longer reachable in the heap, its destructor
+will be invoked by the garbage collection, which can happen after the `Environment`
+is already gone, or after any object it references is gone. To ensure safety,
+the cleanups of a cppgc-managed object should adhere to different patterns,
+depending on what it needs to do:
+
+1. If it does not need to do any non-trivial, nor does its members, just use
+   the default destructor.
+2. If the cleanup relies on a living `Environment`, the class should use this
+   pattern in its class body:
```

Comment:
This term "living `Environment`" is not defined. What is a living environment?

Reply:
In this context, the term "living Enviroment" refers to a condition where a cppgc object can out-live its `Environment` as the cppgc heap is cleaned up after the `Environment` been destructed, as documented above:

> When a cppgc-managed object is no longer reachable in the heap, its destructor
will be invoked by the garbage collection, which can happen after the `Environment`
is already gone.

Though, it didn't mention the term "living Environment".

Reply:
Yeah, I know what was meant, but it should be explained a bit more in the document so that it is clear.